solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/List/Cell/RepositoryCellViewConfiguration.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  RepositoryCellViewConfiguration.swift
+//  OnboardingChallenge
+//
+//  Created by Mateus Nazário Coelho on 16/12/21.
+//
+
+import Foundation
+
+struct RepositoryCellViewConfiguration {
```

Comment:
[nitpick - you can ignore this comment if you want] As a personal preference, when the `Configuration` objects are this small, I prefer to have them in the same file of the view that is going to use them. It avoids jumping from one file to another and can save some time... but it's a personal preference.